rest/handler.go

```diff
@@ -477,10 +477,10 @@ func checkAdminAuth(bucketName, basicAuthUsername, basicAuthPassword string, att
 	if bucketName != "" {
 		requestRoles = BucketScopedEndpointRoles
 	} else {
-		if attemptedHTTPOperation == http.MethodPut || attemptedHTTPOperation == http.MethodPost {
-			requestRoles = ClusterScopedEndpointRolesWrite
-		} else {
+		if attemptedHTTPOperation == http.MethodGet || attemptedHTTPOperation == http.MethodHead {
```

Comment:
Might want `http.MethodOptions` to be considered a read-only op. It's used (or allowed) in a couple of places in SG, but not explicitly in the routing handler list.

Reply:
Added